packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -649,10 +659,66 @@ class DefaultMutationMapperStore implements MutationMapperStore {
     );
 
     readonly ptmData: MobxPromise<PostTranslationalModification[]> = remoteData(
+        {
+            await: () => [this.uniprotPtmData],
```

Comment:
Do we need to wait for `dbPtmData` here?

Reply:
Nice catch! We should probably only wait for the PTM source set with `this.ptmSource`.